client/me/purchases/utils.js

```diff
@@ -66,14 +66,18 @@ function recordPageView( trackingSlug, props, nextProps = null ) {
 
 	const { productSlug } = purchase;
 
-	analytics.tracks.recordEvent( `calypso_${ trackingSlug }_purchase_view`, { product_slug: productSlug } );
+	analytics.tracks.recordEvent( `calypso_${ trackingSlug }_purchase_view`, {
+		product_slug: productSlug,
+	} );
 }
 
 function canEditPaymentDetails( purchase ) {
 	if ( ! config.isEnabled( 'upgrades/credit-cards' ) ) {
 		return false;
 	}
-	return ! isExpired( purchase ) && ! isOneTimePurchase( purchase ) && ! isIncludedWithPlan( purchase );
+	return (
```

Comment:
I bet line was too long, so it had to wrap with the parenthesis to avoid syntax error when splitting line.